modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Instructions.kt

```diff
@@ -93,9 +95,18 @@ object Instructions {
     ): Instruction.SetKeyValue {
         return Instruction.SetKeyValue(
             SetKeyValueBox(
-                objectId = EvaluatesTo(Expression.Raw(Value.Id(IdBox.AssetId(assetId)))),
-                key = EvaluatesTo(Expression.Raw(key.asValue())),
-                value = EvaluatesTo(Expression.Raw(value))
+                objectId = IdBox.AssetId(assetId).evaluatesTo(),
+                key = key.evaluatesTo(),
+                value = value.evaluatesTo()
+            )
+        )
+    }
+
+    fun removeAsset(assetId: AssetId, key: String): Instruction.RemoveKeyValue {
```

Comment:
This name is ambiguous. I think better to rename it. Also need to rename `storeAsset` as well